flake.nix

```diff
@@ -75,6 +75,11 @@
             };
           };
         };
+        devShells.default = pkgs.mkShell {
+          packages = with pkgs; [
+            lua-language-server
```

Comment:
TBH, I'm not sure if this is actually needed. While I'm not able to run the `lua-language-server` executable in the dev environment before or after this update. Without this, the lua_ls server is still able to attach to the client. This may be because of mason. If that's the case it brings into question if I bring into the language specific lsp servers in my nix dev templates. 🤔